Cargo.toml

```diff
@@ -19,13 +19,15 @@ prost-derive = "0.4"
 serde = "1.0"
 serde_derive = "1"
 structopt = "0.2"
-navitia_model = { git = "https://github.com/CanalTP/navitia_model.git", rev = "724b1b3d77ce9871cb02ff87c2d78fe64b2b27c0" }
+navitia_model = { git = "https://github.com/antoine-de/navitia_model.git", rev = "a28c1dc19cbc1fa137c90c169a14b9285cfbbf6a" }
 chrono-tz = "0.5"
 serde_yaml = "0.8"
 
 [dev-dependencies]
 serde_json = "1.0"
 mockito = "0.14"
+structopt = "0.2"
```

Comment:
C’est normal de les avoir qu’en dépedance de dev ?

Pas sûr de comprendre le `model-builder` qui est le même que navitia_model. Ça permet d’avoir autre chose ?

Reply:
oups structopt was a mistake (I don't really know how this was inserted...)

the model-builder is a sub crate of navitia-model, this will makes it possible to have different release path for both library.